styles/style.css

```diff
@@ -0,0 +1,133 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+.cf:before,
+.cf:after {
+  content: "";
+  display: table;
+}
+
+.cf:after {
+  clear: both;
+}
+.container {
+  max-width: 960px;
+  margin: 0 auto;
+}
+header {
+  position: relative;
+  width: 100%;
+  height: 80px;
+  background-color: #EA4335;
+}
+
+.logo {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 80px;
+  height: 80px;
+  background-color: #3375DA;
+}
+
+.main-nav {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 70%;
+  height: 80px;
+  background-color: #34A853;
+}
+
+.main-nav li {
+  float: left;
```

Comment:
Since you are floating your list items, its parent element, `<ul>`, needs to be clear its children's floats. 

This isn't appearing as a visual issue in your web design because you are setting a static height of `80px` on your `.main-nav` element, but you can see what I'm saying if you try to apply a `background-color` to the `ul` element and don't see anything happen (see screenshot below). 

![screen shot 2015-09-19 at 3 01 47 pm](https://cloud.githubusercontent.com/assets/651702/9978728/d4e337f4-5edf-11e5-984a-52e7f69b1389.png)